include/extractor/scripting_environment_lua.hpp

```diff
@@ -49,8 +52,8 @@ struct LuaScriptingContext final
 class Sol2ScriptingEnvironment final : public ScriptingEnvironment
 {
   public:
-    static const constexpr int SUPPORTED_MIN_API_VERSION = 0;
-    static const constexpr int SUPPORTED_MAX_API_VERSION = 1;
+    static const constexpr int SUPPORTED_MIN_API_VERSION = 1;
```

Comment:
to clarify, setting version=0 was treated the same way as if no version was set in the profile file. i agree we should keep supporting both cases: no version set and version specifically set to 0. of course also version set to 1.